proto/qdrant/cloud/quota/v1/quota.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package qdrant.cloud.quota.v1;
+
+import "buf/validate/validate.proto";
+import "google/api/annotations.proto";
+import "qdrant/cloud/common/v1/common.proto";
+
+// Quota is the API used to configure Quotas.
+service QuotaService {
+  // Lists all quotas for the authenticated user.
+  // Required permissions:
+  // - None (authenticated only)
+  rpc ListAuthenticatedUserQuotas(ListAuthenticatedUserQuotasRequest) returns (ListAuthenticatedUserQuotasResponse) {
+    // permissions
+    option (common.v1.permissions) = "";
+    // custom account-id expression
+    option (qdrant.cloud.common.v1.account_id_expression) = "";
+    // custom supported-actor-types expression. (Requires to be authenticated as a user)
+    option (qdrant.cloud.common.v1.supported_actor_types) = ACTOR_TYPE_USER;
+    // gRPC Gateway REST call
+    option (google.api.http) = {get: "/api/quota/v1/users/me/quotas"};
+  }
+  // Lists all quotas for the account identified by the given account ID.
+  // Required permissions:
+  // - read:account
+  rpc ListAccountQuotas(ListAccountQuotasRequest) returns (ListAccountQuotasResponse) {
+    // permissions
+    option (common.v1.permissions) = "read:account";
+    // gRPC Gateway REST call
+    option (google.api.http) = {get: "/api/quota/v1/accounts/{account_id}/quotas"};
+  }
+}
+
+// buf:lint:ignore QDRANT_CLOUD_REQUIRED_REQUEST_FIELDS
+// ListAuthenticatedUserQuotasRequest is the request for the ListAuthenticatedUserQuotas function
+message ListAuthenticatedUserQuotasRequest {
+  // Empty
+}
+
+// ListAuthenticatedUserQuotasResponse is the response from the ListAuthenticatedUserQuotas function
+message ListAuthenticatedUserQuotasResponse {
+  // ID of the user this quota applies to
+  string user_id = 1;
```

Comment:
I assume you know who you are, we are not adding this to other responses either, so imo we should drop it here.